unison-src/transcripts/fix1926.md

```diff
@@ -0,0 +1,15 @@
+```ucm
+.> builtins.merge
```

Comment:
Sometimes I'll just use an `alias.type ##Nat Nat` instead of a full `builtins.merge`, but I didn't wanna modify it in this case.